libs/server/Storage/Session/StorageSession.cs

```diff
@@ -65,7 +66,7 @@ public StorageSession(StoreWrapper storeWrapper,
             functionsState = storeWrapper.CreateFunctionsState();
 
             var functions = new MainSessionFunctions(functionsState);
-            var session = storeWrapper.store.NewSession<SpanByte, SpanByteAndMemory, long, MainSessionFunctions>(functions);
+            var session = storeWrapper.store.NewSession<RawStringInput, SpanByteAndMemory, long, MainSessionFunctions>(functions);
```

Comment:
The storage session should keep a single parseState allocated at its constructor time, similar to RespServerSession, so that we do not allocate per call.